src/pages/Profile/Profile.tsx

```diff
@@ -1,10 +1,17 @@
 import UserPosts from "@/pages/Profile/components/UserPosts/UserPosts"
 import UserProfile from "@/pages/Profile//components/UserProfile/UserProfile"
+import { useParams } from "react-router-dom"
+import Navbar from "@/components/Navbar/Navbar"
+import useProfile from "./hooks/useProfile"
 
 const Profile = () => {
+  const { id } = useParams()
+  const { isMyPage } = useProfile({ userId: id })
+
   return (
     <>
```

Comment:
재영님 말씀대로 컴포넌트의 최상위에 layout이 존재하면 좋을 것 같아요!! 😊
```suggestion
    <UserProfileLayout>
```
네이밍은 어떻게 해야 할지 잘 모르겠지만 이런 느낌으로 해보시는 건 어떨까요?🤔